src/sentry/tasks/digests.py

```diff
@@ -39,7 +40,11 @@ def deliver_digest(key, schedule_timestamp=None):
     from sentry.app import digests
 
     plugin, project = split_key(key)
-    with digests.digest(key) as records:
+    minimum_delay = ProjectOption.objects.get_value(
+        project,
+        '{0}:digests:{1}'.format(plugin.get_conf_key(), 'minimum_delay'),
+    )
```

Comment:
I figured that it would be a safe assumption that these values were actually integers, since [`ProjectOption.value` is a `UnicodePickledObjectField`](https://github.com/getsentry/sentry/blob/master/src/sentry/models/projectoption.py#L117).